pkg/cmd/gtctl/cluster/create/create.go

```diff
@@ -80,13 +82,13 @@ func NewCreateClusterCommand(l logger.Logger) *cobra.Command {
 				clusterName = args[0]
 				ctx         = context.Background()
 				cancel      context.CancelFunc
+				delOpt      component.DeleteOptions
```

Comment:
nitpick: maybe `deleteOpts` or `delOpts` will be better naming.

Reply:
updated